.changeset/brown-lies-wonder.md

```diff
@@ -0,0 +1,28 @@
+---
+"@aws-amplify/ui-react": patch
+"@aws-amplify/ui": patch
+"@aws-amplify/ui-angular": patch
+"@aws-amplify/ui-vue": patch
+---
+
+(Internal) Use Amplify JS' new `autoSignIn` flag to do automatic sign in. We were doing this manually previously. 
+
```

Comment:
Nit: I think we can delete this now, since you added public facing descriptions!

```suggestion
```

Reply:
Ok deleted